src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -96,12 +100,12 @@ const SkillsQuizStepper = () => {
           title="Skills Quiz"
           className="bg-light-200"
           isOpen
-          onClose={() => console.log('Skills quiz closed.')}
+          onClose={() => closeSkillsQuiz()}
```

Comment:
Just a question. what is the difference between the below two?

`onClose={() => closeSkillsQuiz()}` and `onClose={closeSkillsQuiz}`

Reply:
I think the first one is the arrow function and latter one is a simple function. Both will work in this case.

Reply:
In both cases, a reference to a function is being passed that will be called when "close" event triggers. In first case, it is passing reference of the parent function that eventually calls our function `closeSkillsQuiz`.  In second case, it's just simply passing reference to our function.